<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKlien</class>
 <widget class="QWidget" name="FormKlien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idklienLabel">
      <property name="text">
       <string>idklien</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idklienLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaklienLabel">
      <property name="text">
       <string>namaklien</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaklienLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamatklienLabel">
      <property name="text">
       <string>alamatklien</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="alamatklienLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telpklienLabel">
      <property name="text">
       <string>telpklien</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="telpklienLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableKlien">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
